allstar.yaml

```diff
@@ -0,0 +1,8 @@
+# https://github.com/ossf/allstar#policies
+optConfig:
+  optOutStrategy: false
+  optOutPrivateRepos: false
+  optOutPublicRepos: false
+  optInRepos:
+    - "opentelemetry-go"
+    - "opentelemtry-go-contrib"
```

Comment:
Should we also include `opentelemetry-go-build-tools` and `opentelemetry-proto-go`?